packages/storybook/stories/Examples/TeamsInterop/snippets/Lobby.snippet.tsx

```diff
@@ -51,21 +52,32 @@ export const Lobby = (props: LobbyProps): JSX.Element => {
 
       <div
         style={{
-          zIndex: 0,
-          textAlign: 'center',
-          width: '50%',
-          height: '25%',
-          overflow: 'none',
-          margin: 'auto',
           position: 'absolute',
-          top: 0,
-          left: 0,
-          bottom: 0,
-          right: 0
+          top: '50%',
+          left: '50%',
+          transform: 'translate(-50%, -50%)',
+          marginTop: '-34px'
```

Comment:
How well does it scale with different Monitor heights?
If my monitor was in portrait mode, would this still be centered?

Reply:
I played around with resizing. Providing some screenshots for context
<img width="612" alt="Screen Shot 2021-05-31 at 1 32 57 PM" src="https://user-images.githubusercontent.com/9044372/120240204-d98d9080-c214-11eb-9be6-64dcf9a6a0c0.png">
<img width="744" alt="Screen Shot 2021-05-31 at 1 33 12 PM" src="https://user-images.githubusercontent.com/9044372/120240207-da262700-c214-11eb-8ad1-027410dc5491.png">

Reply:
Does it stay centered when browser height changes?

Reply:
Yep I played around with the browser sizes and everything looks good